instance/tests/models/test_instance.py

```diff
@@ -705,6 +705,7 @@ def test_provision_failed_exception_email(self):
         self.assertEqual(mime_type, "text/html")
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
Can we address that comment instead of disabling it, especially at the global level on that file? pylint warns for a good reason here. During the pylint upgrade I've noticed that we have taken this shortcut a few times throughout the codebase, and created a ticket to address this (OC-1572). It would be good to not add to that debt.